internal/rbd/rbd_util.go

```diff
@@ -96,6 +96,11 @@ type rbdImage struct {
 	// VolSize is the size of the RBD image backing this rbdImage.
 	VolSize int64
 
+	// image striping configurations.
+	StripCount uint64
```

Comment:
Here and everywhere else in the PR, including comments:
```suggestion
	StripeCount uint64
```